src/importexport/guitarpro/internal/gtp/gpconverter.cpp

```diff
@@ -58,72 +58,72 @@
 using namespace mu::engraving;
 
 namespace mu::engraving {
-static Jump::Type jumpType(const QString& typeString)
-{
-    static QMap<QString, Jump::Type> types {
-        { "DaCapo", Jump::Type::DC },
-        { "DaCapoAlFine", Jump::Type::DC_AL_FINE },
-        { "DaCapoAlCoda", Jump::Type::DC_AL_CODA },
-        { "DaCapoAlDoubleCoda", Jump::Type::DC_AL_DBLCODA },
-        { "DaSegnoAlCoda", Jump::Type::DS_AL_CODA },
-        { "DaSegnoAlDoubleCoda", Jump::Type::DS_AL_DBLCODA },
-        { "DaSegnoAlFine", Jump::Type::DS_AL_FINE },
-        { "DaSegnoSegno", Jump::Type::DSS },
-        { "DaSegnoSegnoAlCoda", Jump::Type::DSS_AL_CODA },
-        { "DaSegnoSegnoAlDoubleCoda", Jump::Type::DSS_AL_DBLCODA },
-        { "DaSegnoSegnoAlFine", Jump::Type::DSS_AL_FINE },
-        { "DaCoda", Jump::Type::DCODA },
-        { "DaDoubleCoda", Jump::Type::DDBLCODA },
-        { "DaCoda", Jump::Type::DCODA },
+static Jump::Type jumpType(const String& typeString)
+{
+    static std::map<String, Jump::Type> types {
+        { String("DaCapo"), Jump::Type::DC },
+        { String("DaCapoAlFine"), Jump::Type::DC_AL_FINE },
+        { String("DaCapoAlCoda"), Jump::Type::DC_AL_CODA },
+        { String("DaCapoAlDoubleCoda"), Jump::Type::DC_AL_DBLCODA },
+        { String("DaSegnoAlCoda"), Jump::Type::DS_AL_CODA },
+        { String("DaSegnoAlDoubleCoda"), Jump::Type::DS_AL_DBLCODA },
+        { String("DaSegnoAlFine"), Jump::Type::DS_AL_FINE },
+        { String("DaSegnoSegno"), Jump::Type::DSS },
+        { String("DaSegnoSegnoAlCoda"), Jump::Type::DSS_AL_CODA },
+        { String("DaSegnoSegnoAlDoubleCoda"), Jump::Type::DSS_AL_DBLCODA },
+        { String("DaSegnoSegnoAlFine"), Jump::Type::DSS_AL_FINE },
+        { String("DaCoda"), Jump::Type::DCODA },
+        { String("DaDoubleCoda"), Jump::Type::DDBLCODA },
+        { String("DaCoda"), Jump::Type::DCODA },
```

Comment:
The better solution (at least the solution that is used in most other places) is to prefix the literals with `u`, i.e. `u"DaCapo"`, `u"DaCapoAlFine"`.